CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * PayPal
   * Fix an issue where `PayPalRequest` was sending `phone_number` instead of `payer_phone`
   * Add `merchant` and `flow_type` as query parameters to the app switch URL.
+  * Add `paymentId` to `PayPalAccountNonce`
```

Comment:
This name is used across the API and web SDK so this maintains consistency across platforms